scripts/update-spectrum-css.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { readFileSync, writeFileSync } from 'fs';
+import latestVersion from 'latest-version';
+import fg from 'fast-glob';
+
+async function update() {
+    let updated = false;
+    for (const packageJSONPath of await fg([
+        `./package.json`,
+        `./packages/**/package.json`,
+        `./projects/**/package.json`,
+        `./tools/**/package.json`,
+    ])) {
```

Comment:
Could this be similarly expressed as:
```suggestion
    for (const packageJSONPath of await fg([
        `**/package.json`
    ], { ignore: ['**/node_modules/**'] })) {
```

Reply:
The benefit here being that it would capture packages even if new workspaces are added or folders renamed.